docs/rancher/cloud-provider.md

```diff
@@ -47,41 +47,92 @@ When spinning up an RKE cluster using the Harvester node driver, you can perform
 1. Select `Harvester(Out-of-tree)` option.
 
     ![](/img/v1.2/rancher/rke-cloud-provider.png)
-  
+
 2. Install `Harvester Cloud Provider` from the Rancher marketplace.
 
     ![](/img/v1.2/rancher/install-harvester-cloud-provider.png)
 
-  
+
 ### Deploying to the RKE2 Cluster with Harvester Node Driver
 
 When spinning up an RKE2 cluster using the Harvester node driver, select the `Harvester` cloud provider. The node driver will then help deploy both the CSI driver and CCM automatically.
 
   ![](/img/v1.2/rancher/rke2-cloud-provider.png)
 
-### Deploying to the RKE2 custom cluster (experimental)
+### Deploying to the RKE2 custom cluster with Harvester Cloud Provider (experimental)
 
 ![](/img/v1.2/rancher/custom.png)
 1. Use `generate_addon.sh` to generate a cloud-config and place it into the directory `/etc/kubernetes/cloud-config` on every custom node.
 
-   ```
-   curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace> 
-   ```
+    ```
+    curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace>
+    ```
+
+    The output will look as follows:
+
+    ```
+    ########## cloud config ############
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: <CACERT>
+        server: https://HARVESTER-ENDPOINT/k8s/clusters/local
+      name: local
+    contexts:
+    - context:
+        cluster: local
+        namespace: default
+        user: harvester-cloud-provider-default-local
+      name: harvester-cloud-provider-default-local
+    current-context: harvester-cloud-provider-default-local
+    kind: Config
+    preferences: {}
+    users:
+    - name: harvester-cloud-provider-default-local
+      user:
+        token: <TOKEN>
+
+    ########## cloud-init user data ############
+    write_files:
+    - encoding: b64
+      content: <CONTENT>
+      owner: root:root
+      path: /etc/kubernetes/cloud-config
+      permissions: '0644'
+    ```
 
-  :::note
+    :::note
```

Comment:
Please remove this. In another comment, I suggested placing the note after the command.